src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int FIGURE_COUNT = 100;
```

Comment:
```suggestion
    private static final int FIGURE_COUNT = 101;
```
this argument is exclusive
![image](https://github.com/mate-academy/jv-lottery/assets/49767049/d4d07f3c-53a9-4419-9677-20ce93a4621c)